js/main.js

```diff
@@ -0,0 +1,3 @@
+$('stopButton').click(illuminateRed);
```

Comment:
@anthonypanayiotou these click functions aren't working because inside the parentheses you are not calling the functions you created, does that make sense? please edit these and those lights will work! :)